sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -551,6 +586,548 @@ Parameters:
+
+It is not an error if `newBufferRef` is set to the current parameter value in any
+registered nodes.
+
+Parameters:
+
+* `newBufferRef` - The new buffer that the dynamic accessor will provide access to
+in all registered graph nodes.
+
+|
+[source,c++]
+----
+template <typename AllocatorT>
+void update(
+        buffer<DataT, Dimensions, AllocatorT> &newBufferRef,
+        range<Dimensions> newAccessRange);
+----
+
+|Updates the buffer that this dynamic accessor provides access to in all graph nodes
+where it is registered. This new value will be reflected immediately in the modifiable
+graph which contains the registered nodes. The new value will not be reflected in any
+executable graphs created from that modifiable graph until `command_graph::update()`
+is called passing the modified nodes, or a new executable graph is finalized from
+the modifiable graph.
+
+It is not an error if `newBufferRef` is set to the current parameter value in any
+registered nodes.
+
+Parameters:
+
+* `newBufferRef` - The new buffer that the dynamic accessor will provide access to
+in all registered graph nodes.
+* `newAccessRange` - Access range for the accessor of the new buffer parameter.
+
+Exceptions:
+
+Throws an exception with the `errc::invalid` error code if `newAccessRange` exceeds
+the range of `bufferRef` in any dimension.
+
+|
+[source,c++]
+----
+template <typename AllocatorT>
+void update(
+        buffer<DataT, Dimensions, AllocatorT> &newBufferRef,
+        range<Dimensions> newAccessRange,
+        id<Dimensions> newAccessOffset);
+----
+
+|Updates the buffer that this dynamic accessor provides access to in all graph nodes
+where it is registered. This new value will be reflected immediately in the modifiable
+graph which contains the registered nodes. The new value will not be reflected in any
+executable graphs created from that modifiable graph until `command_graph::update()`
+is called passing the modified nodes, or a new executable graph is finalized from
+the modifiable graph.
+
+It is not an error if `newBufferRef` is set to the current parameter value in any
+registered nodes.
+
+Parameters:
+
+* `newBufferRef` - The new buffer that the dynamic accessor will provide access to
+in all registered graph nodes.
+* `newAccessRange` - Access range for the accessor of the new buffer parameter.
+* `newAccessOffset` - Access offset for the range of the new buffer parameter.
+
+Exceptions:
+
+Throws an exception with the `errc::invalid` error code if the sum of
+`newAccessRange` and `newAccessOffset` exceeds the range of `bufferRef` in any dimension.
+
+|
+[source,c++]
+----
+accessor<DataT, Dimensions, AccessMode, AccessTarget> get(handler &cgh);
+----
+
+| Returns a `sycl::accessor` to use in the current command-group.
+The accessor is registered with this `dynamic_accessor` instance
+and will be updated whenever `dynamic_accessor::update()` is called.
+
+Parameters:
+
+* `handler` - The kernel handler that represents the current submission.
```

Comment:
param is called `cgh` rather than `handler` (same comment for other dynamic classes)